featuretools/primitives/standard/datetime_transform_primitives.py

```diff
@@ -331,6 +331,39 @@ def is_leap_year(vals):
         return is_leap_year
 
 
+class IsLunchTime(TransformPrimitive):
+    """Determines if a datetime falls during configurable lunch hour
+
+    Args:
+        lunch_hour (int): Time when lunch is taken. Defaults to 12
```

Comment:
Maybe we should specify what are valid values here. For example, to set 1pm as the lunch hour, I assume users should pass in `13`?